modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -22,154 +25,89 @@ For constraints that are backed by an index, the index provider for the backing
 Only one valid value exists for the index provider, `range-1.0`, which is the default value.
 There is no supported index configuration for range indexes.
 
-[NOTE]
-====
-Creating a constraint requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-constraints[`CREATE CONSTRAINT` privilege].
-====
+[[create-property-uniqueness-constraints]]
+=== Create property uniqueness constraints
 
-[[constraints-syntax-create-node-unique]]
-=== Node property uniqueness constraint
-
-This command creates a property uniqueness constraint on nodes with the specified label and properties.
-
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a node uniqueness constraint on a single property
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR (n:LabelName)
 REQUIRE n.propertyName IS [NODE] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a composite node uniqueness constraint on multiple properties
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR (n:LabelName)
 REQUIRE (n.propertyName_1, ..., n.propertyName_n) IS [NODE] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-Index provider can be specified using the `OPTIONS` clause.
-
-[role=label--new-5.7]
-[[constraints-syntax-create-rel-unique]]
-=== Relationship property uniqueness constraint
-
-This command creates a property uniqueness constraint on relationships with the specified relationship type and properties.
-
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a relationship uniqueness constraint on a single property label:new[Introduced in 5.7]
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR ()-"["r:RELATIONSHIP_TYPE"]"-()
 REQUIRE r.propertyName IS [REL[ATIONSHIP]] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-[source, syntax, role="noheader", indent=0]
+.Syntax for creating a composite relationship uniqueness constraint on multiple properties label:new[Introduced in 5.7]
+[source, syntax]
 ----
 CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
 FOR ()-"["r:RELATIONSHIP_TYPE"]"-()
 REQUIRE (r.propertyName_1, ..., r.propertyName_n) IS [REL[ATIONSHIP]] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-Index provider can be specified using the `OPTIONS` clause.
```

Comment:
this sentence was removed for the property uniqueness constraint but kept and updated to
````
An index provider can be specified using the `OPTIONS` clause.
```
in the key constraint section